src/org/zaproxy/zap/extension/pscanrulesAlpha/ContentSecurityPolicyMissingScanner.java

```diff
@@ -138,16 +162,20 @@ public String getName(){
 		return Constant.messages.getString(MESSAGE_PREFIX + "name");
 	}
 	
-	private String getDescription() {
-		return Constant.messages.getString(MESSAGE_PREFIX + "desc");
+	public String getName(String sub) {

```

Comment:
Probably better to create a generic `getMessage(String)` method instead, the implementation is the same for all methods (getName/getDescription/getSolution/...) so no need to have different methods.